quasi_macros/src/lib.rs

```diff
@@ -419,11 +419,11 @@ fn expand_tts(cx: &ExtCtxt, sp: Span, tts: &[ast::TokenTree])
     p.quote_depth += 1;
 
     let cx_expr = p.parse_expr();
-    if !p.eat(&token::Comma) {
-        p.fatal("expected token `,`");
+    if !p.eat(&token::Comma).ok().unwrap() {
```

Comment:
That is the behavior from before. The issue is that `Display` isn't implemented for the error type. But `FatalError` is simply an empty struct -> nothing to actually display.